### PR TITLE
[AMORO-1892] Improve the SQL Shortcuts in Terminal Web UI

### LINKED ISSUE #1892: [Improvement]: Improve the SQL Shortcuts in Terminal Web UI
### Search before asking

- [X] I have searched in the [issues](https://github.com/NetEase/amoro/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

Currently, the SQL Shortcuts feature is not user-friendly because it overrides all SQLs in the terminal, causing previous SQLs to be lost.

image

### How should we improve?

Maybe we should append the new SQL demo to the last line instead of overriding all the SQLs.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Subtasks

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by tcodehuber**
## Why are the changes needed?

Close #1892.

## Brief change log

image

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java [hunk 1]
```diff
@@ -36,10 +36,13 @@ public enum SqlExample {
       "alter table db_name.table_name add column data int ;\n"
           + "alter table db_name.table_name alter column data bigint ;\n"
           + "alter table db_name.table_name drop column data;"),
-  ALTER_PROPERTIES(
-      "AlterProperties",
+  ADD_PROPERTIES(
+      "AddProperties",
```
Comment (zhoujinsong) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java:40]: Is `SetProperties` proper here? the SQL can both add and update properties.

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java [hunk 2]
```diff
@@ -36,10 +36,13 @@ public enum SqlExample {
...
       "alter table db_name.table_name set tblproperties (\n"
           + "    'comment' = 'A table comment.');"),
+  DROP_PROPERTIES(
+      "DropProperties",
```
Comment (zhoujinsong) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java:44]: If we use `SetProperties`, maybe we should use `UnsetProperties` here.

**Review by zhoujinsong — APPROVED** [review]
@tcodehuber I left some comments.

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java (hunk 1 above)
Comment (tcodehuber) [reply ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java:40]: Yeah, more reasonable.

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java (hunk 2 above)
Comment (tcodehuber) [reply ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java]: Plz review it again.

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Comment by wangtaohz** [comment]
Thank you for your contribution. 👍 It's a very helpful improvement.

**Pull request merged**